main.py

```diff
@@ -7,17 +7,12 @@
 import clipboard
 
 database = db.DB("pugsey.db")
-logged_in_user = None
-user_info = None
```

Comment:
You're using `user_info` now so add this line back. It should be declared as a global variable